lsl/common/stations.py

```diff
@@ -342,26 +334,11 @@ def get_enz_offset(self, locTo):
         in meter along the east, north, and vertical directions.
         """
         
-        ecefFrom = self.geocentric_location
-        try:
-            ecefTo = locTo.geocentric_location
-        except AttributeError:
-            ecefTo = geo_to_ecef(float(locTo[0])*numpy.pi/180, float(locTo[1])*numpy.pi/180, locTo[2])
-            
-        ecefFrom = numpy.array(ecefFrom)
-        ecefTo = numpy.array(ecefTo)
-        
-        rho = ecefTo - ecefFrom
-        rot = numpy.array([[ numpy.sin(self.lat)*numpy.cos(self.long), numpy.sin(self.lat)*numpy.sin(self.long), -numpy.cos(self.lat)], 
-                        [-numpy.sin(self.long),                     numpy.cos(self.long),                      0                  ],
-                        [ numpy.cos(self.lat)*numpy.cos(self.long), numpy.cos(self.lat)*numpy.sin(self.long),  numpy.sin(self.lat)]])
-        sez = numpy.dot(rot, rho)
         
-        # Convert from south, east, zenith to east, north, zenith
-        enz = 1.0*sez[[1,0,2]]
-        enz[1] *= -1.0
-        
-        return enz
+        az, alt, dist = self.get_pointing_and_distance(locTo)
+        return np.array([np.sin(az)*np.cos(alt),
+                         np.cos(az)*np.cos(alt),
+                         np.sin(alt)])*dist
```

Comment:
This is one area where we need to check to make sure that we are getting identical answers.

Reply:
This is now tested and in agreement with what was put in.